### PR TITLE
WIP: Docs for components

### PR DESCRIPTION
**Pull request by shivampandey0**
- added home page.
- Installation page.
- alerts page.

## Implemented `Reset` feature for alerts as said in code reviews.

https://user-images.githubusercontent.com/51923995/153407438-db9f3a49-0964-459f-a3b1-78f4ec9b6e78.mov

### CODE REVIEW

File: docs/alerts.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://marvel-ui.vercel.app/css/marvel-ui.css" />
+    <link rel="stylesheet" href="./styles/docs.css" />
+    <title>Get Started | Marve UI</title>
```
Comment (MaruthiPothuganti) [inline docs/alerts.html]: Typo: Marvel

**Review by AnuragSahu11 — APPROVED** [review]
Looks good

**Review by MaruthiPothuganti — APPROVED** [review]
Looks good to merge👍

**Comment by shivampandey0** [comment]
Fixed typos.

**Pull request merged**